From stable-bounces@linux.kernel.org  Wed Mar 12 14:26:42 2008
Message-ID: <47CDBA77.2010707@cryonet.de>
Date: Tue, 04 Mar 2008 22:09:11 +0100
From: Sven Andersen <s.andersen@cryonet.de>
To: gregkh@suse.de
Cc: stable@kernel.org
Subject: USB: ftdi_sio - really enable EM1010PC

[upstream commit: 4ae897df]

Add EM1010PC to ftdi_sio.c

Signed-off-by: Sven Andersen <s.andersen@cryonet.de>
Cc: stable <stable@kernel.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
Signed-off-by: Chris Wright <chrisw@sous-sol.org>
---

Peter Stark submitted a patch for enabling multiple ELV-devices
and adding the EM1010PC to ftdi_sio which has already been applied.
For some reason this line got lost somewhere.

As Peters patch went into 2.6.24-stable this might be applicable
there too.

---
 drivers/usb/serial/ftdi_sio.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/usb/serial/ftdi_sio.c
+++ b/drivers/usb/serial/ftdi_sio.c
@@ -499,6 +499,7 @@ static struct usb_device_id id_table_com
 	{ USB_DEVICE(FTDI_VID, FTDI_ELV_FS20SIG_PID) },
 	{ USB_DEVICE(FTDI_VID, FTDI_ELV_WS300PC_PID) },
 	{ USB_DEVICE(FTDI_VID, FTDI_ELV_FHZ1300PC_PID) },
+	{ USB_DEVICE(FTDI_VID, FTDI_ELV_EM1010PC_PID) },
 	{ USB_DEVICE(FTDI_VID, FTDI_ELV_WS500_PID) },
 	{ USB_DEVICE(FTDI_VID, LINX_SDMUSBQSS_PID) },
 	{ USB_DEVICE(FTDI_VID, LINX_MASTERDEVEL2_PID) },
